ow_plexil/src/plans/owlat_plans/PrintNodeStart.plp

```diff
@@ -0,0 +1,11 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "owlat-interface.h"
+
+PrintNodeStart:
+{
+  In String NodeName;
+  log_info("Starting ", NodeName, "...");
```

Comment:
A blank line before this would make it easier to separate the transcripts of consecutive plan runs.